### PR TITLE
feat(content): Adjust Firelight description

### PR DESCRIPTION
**Pull request by Saugia**
**Content (Artwork / Missions / Jobs)**

## Summary
Someone brought up that the first line in the Firelight description came off as a bug because it referenced the Firestorm. This isn't a bug and is intended, and the description is lore-based, but it got me thinking that if we mention the Firestorm in the description, especially as one of the first few words, we should probably also mention the Firelight to avoid confusion. This PR adds a quick blurb at the end of the first paragraph that does so.

I'm not attached to this, so if this isn't desired, I don't mind closing it or editing it in a different manner.

## Save File
N/A

### CODE REVIEW

**Review by Zitchas — APPROVED** [review]
This seems like a reasonable addition to ensure people know that the reference is deliberate.

File: data/korath/korath weapons.txt [hunk 1]
```diff
@@ -952,7 +952,7 @@ outfit "Firelight Missile Bank"
 		"missile strength" 60
 		submunition "Firelight Turning"
 		"stream"
-	description "As the Firestorm torpedo does not require full reactor shielding or safety interlocks, Korath engineers were able to further scale down the reactor element to fit in a missile casing for use on and against smaller vessels."
+	description "As the Firestorm torpedo does not require full reactor shielding or safety interlocks, Korath engineers were able to further scale down the reactor element to fit in a missile casing for use on and against smaller vessels. These newer, more compact Firestorms, are called Firelight missiles."
```
Comment (EjoThims) [inline data/korath/korath weapons.txt:955]:
```suggestion
	description "As the Firestorm torpedo does not require full reactor shielding or safety interlocks, Korath engineers were able to further scale down the reactor element to fit in a missile casing for use on and against smaller vessels. This newer, more compact version is called the Firelight missiles."
```

**Review by EjoThims — CHANGES_REQUESTED** [review]
Just to avoid mentioning the other one more often still :)

**Review by EjoThims — APPROVED** [review]

**Review by tibetiroka — APPROVED** [review]

File: data/korath/korath weapons.txt [hunk 2]
```diff
@@ -952,7 +952,7 @@ outfit "Firelight Missile Bank"
 		"missile strength" 60
 		submunition "Firelight Turning"
 		"stream"
-	description "As the Firestorm torpedo does not require full reactor shielding or safety interlocks, Korath engineers were able to further scale down the reactor element to fit in a missile casing for use on and against smaller vessels."
+	description "As the Firestorm torpedo does not require full reactor shielding or safety interlocks, Korath engineers were able to further scale down the reactor element to fit in a missile casing for use on and against smaller vessels. This newer, more compact version is called the Firelight missile."
```
Comment (ravenshining) [inline data/korath/korath weapons.txt:955]: That blurb is just is just redundant on several levels.  Here:

```suggestion
	description "As the Firestorm torpedo does not require full reactor shielding or safety interlocks, Korath engineers were able to create the Firelight by further scaling down the reactor element to fit in a missile casing for use on and against smaller vessels."
```

**Review by ravenshining — CHANGES_REQUESTED** [review]

File: data/korath/korath weapons.txt (hunk 2 above)
Comment (Saugia) [reply data/korath/korath weapons.txt]: Yeah, definitely reads better.

**Pull request merged**